src/world/partners.c

```diff
@@ -621,7 +655,31 @@ INCLUDE_ASM(void, "world/partners", partner_walking_follow_player, Npc* partner)
 
 INCLUDE_ASM(void, "world/partners", partner_flying_enable, Npc* partner, s32 val);
 
-INCLUDE_ASM(void, "world/partners", partner_flying_update_player_tracking, Npc* partner);
+void partner_flying_update_player_tracking(Npc* partner) {
+    PlayerStatus* playerStatus = &gPlayerStatus;
+    struct8010CD38* tempStruct;
+    f32 tempY;
+    s32 Zero = 0; // Why
```

Comment:
Why indeed... `|| zero == 0` below very much looks like a fakematch, perhaps add a TODO to that effect to have someone take a look at it later.

Also call this `zero`, not `Zero`.

Reply:
Possibly `flags`, since this game tends to have flags be the unk_00 of every single struct 😜